### ISSUE TITLE
Post everyday to a Twitter account

### ISSUE THREAD

**Issue by conradfr** [aerosol/fotd#3]
So you can see the function of the day and visit the site if interested.

**Comment by aerosol** [comment]
Yeah I've been thinking about that and not really feeling like doing this for the following reasons:

- That would require someone to run a twitter account - I'd rather pass on that one
- We'd have to track what has been posted in order to avoid dupes (persistence is an extra cost, currently running without it)
- We'd have to implement a client running in the background, deal with failures, somewhat precise intervals, deal with restarts and somewhat ensure that what has been just tweeted is currently on the site
- Perhaps permalinks support would be nice in order to link back to the site (this is minor, we could link directly to hexdocs instead)

It could be a separate project with shared interface for code mangling.
I'd be happy to help if you feel like tackling it. As a start page it is now doing just fine. I might add a "Tweet" button instead.

**Comment by aerosol** [comment]
Added a tweet button
